config.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('lodash');
+var meConf = null;
+
+try {
+    meConf = require('./meConf');
```

Comment:
we talked about allowing to override me.json location with environment variable. need to add it.